proposals/NNNN-byte-address-buffer-alignment.md

```diff
@@ -0,0 +1,983 @@
+---
+title: "NNNN - ByteAddressBuffer Alignment
+params:
+  authors:
+     - mapodaca-nv: Mike Apodaca
+   sponsors:
+     - tex3D: Tex Riddell
+   status: Under Consideration
+---
+
+* Required Version: Shader Model X.Y, Vulkan X.Y, and/or HLSL 20XY
```

Comment:
I suppose gating this on shader model could be required if drivers mishandle `AnnotateHandle` alignment field or different alignments in raw load/store DXIL ops than they've seen or been tested with before.